BackEnd/src/main/java/com/unosquare/admin_core/back_end/configuration/mappings/presentation/RejectedHolidayMappings.java

```diff
@@ -1,44 +1,40 @@
-
 package com.unosquare.admin_core.back_end.configuration.mappings.presentation;
 
-import com.unosquare.admin_core.back_end.viewModels.holidays.CreateHolidayViewModel;
+import com.unosquare.admin_core.back_end.viewModels.holidays.RejectHolidayViewModel;
 import com.unosquare.admin_core.back_end.configuration.mappings.BaseMappings;
 import com.unosquare.admin_core.back_end.configuration.mappings.converters.EventStatusesConverter;
 import com.unosquare.admin_core.back_end.configuration.mappings.converters.EventTypesConverter;
 import com.unosquare.admin_core.back_end.dto.EventDTO;
 import com.unosquare.admin_core.back_end.enums.EventStatuses;
 import com.unosquare.admin_core.back_end.enums.EventTypes;
-import lombok.NoArgsConstructor;
 import org.modelmapper.PropertyMap;
 
 import java.time.LocalDate;
 
-@NoArgsConstructor
-public class SaveHolidayMappings implements BaseMappings<EventDTO, CreateHolidayViewModel> {
+public class RejectedHolidayMappings implements BaseMappings<EventDTO, RejectHolidayViewModel> {
 
     @Override
-    public PropertyMap<EventDTO, CreateHolidayViewModel> MapFromSourceToTarget() {
-        return new PropertyMap<EventDTO, CreateHolidayViewModel>() {
+    public PropertyMap<EventDTO, RejectHolidayViewModel> MapFromSourceToTarget() {
+        return new PropertyMap <EventDTO, RejectHolidayViewModel>() {
             protected void configure() {
+
             }
         };
     }
 
     @Override
-    public PropertyMap<CreateHolidayViewModel, EventDTO> MapFromTargetToSource() {
-        return new PropertyMap <CreateHolidayViewModel, EventDTO>() {
+    public PropertyMap<RejectHolidayViewModel, EventDTO> MapFromTargetToSource() {
+        return  new PropertyMap <RejectHolidayViewModel, EventDTO>() {
             protected void configure() {
-
                 EventTypesConverter eventTypesConverter = new EventTypesConverter();
                 EventStatusesConverter eventStatusConverter = new EventStatusesConverter();
 
                 using(eventTypesConverter.MapFromSourceToTarget()).map(EventTypes.ANNUAL_LEAVE, destination.getEventTypeId());
-                using(eventStatusConverter.MapFromSourceToTarget()).map(EventStatuses.AWAITING_APPROVAL, destination.getEventStatusId());
-                skip().setStartDate(null);
-                skip().setEndDate(null);
-                map().setDateCreated(LocalDate.now());
+                using(eventStatusConverter.MapFromSourceToTarget()).map(EventStatuses.REJECTED, destination.getEventStatusId());
+                map().setEventId(source.getEventId());
```

Comment:
Is this required? Maybe it should be skip instead so we're not doing unnecessary mappings.